packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -206,9 +209,15 @@ const DialoguesList = ({ classes }: { classes: ClassesType<typeof styles> }) =>
   const { PostsItem, SectionButton, SettingsButton, LWTooltip, SingleColumnSection, SectionTitle, SectionSubtitle, DialoguesSectionFrontpageSettings, DialogueRecommendationRow, Typography } = Components
   const currentUser = useCurrentUser()
   const [showSettings, setShowSettings] = useState(false);
+  const { captureEvent } = useTracking();
   const showReciprocityRecommendations = (useABTest(frontpageDialogueReciprocityRecommendations) === "show")
   const showTopics = (useABTest(showTopicsInReciprocity) === "show")
 
+  const { mutate: updateDialogueCheck } = useUpdate({
+    collectionName: 'DialogueChecks',
+    fragmentName: 'DialogueCheckInfo',
+  })
+
```

Comment:
This is unused?

Reply:
outdated, not sure what happened. I fixed in the clean-up